x-pack/plugin/sql/connectors/tableau/README.md

```diff
@@ -0,0 +1,19 @@
+# Tableau connector for Elasticsearch
+
+The Tableau Connector works in tandem with the Elastic JDBC driver to facilitate the query of Elasticsearch. It gives users a simple way to query Elasticsearch data from Tableau.
+After providing basic connection and authentication information, users can easily select Elasticsearch indices for use in Tableau Desktop and Tableau Server.
+
+## Installation
+
+1. Tableau connector for Elasticsearch installation:
+ - Go to the [Connector Download](https://www.elastic.co/downloads/tableau-connector) page.
+ - Download the _.taco_ connector file.
+ - Move the _.taco_ file here:
+    - Windows: C:\Users\[Windows User]\Documents\My Tableau Repository\Connectors
+    -  macOS: /Users/[user]/Documents/My Tableau Repository/Connectors
+2. Elasticsearch JDBC Driver Installation:
+ - Go to the [Driver Download](https://www.elastic.co/downloads/jdbc-client) page.
+ - Download the Elasticsearch JDBC Driver _.jar_ file and move it into the following directory:
```

Comment:
Should we mention something about appropriate versions based on the es node/cluster version?

Reply:
That's a good point. I would however follow up on this:
* once the connector releasing is hooked into the RM (and we'll then easily be able to pair the connector with the driver, version-wise);
* with a doc PR stating our drivers' compatibility policy that we can refer to (from here, among other places).